apps/designer/app/designer/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -367,7 +368,7 @@ export const CssValueInput = ({
   );
 };
 
-const CssValueInputIconButton = styled(TextFieldIconButton, {
+const CssValueInputIconButton = styled(TextFieldIcon, {
```

Comment:
@natlusyht do you remember why you used icon button instead of icon? Was it supposed to be interactive

Reply:
Seems like it should be an icon indeed